python/source/create.rst

```diff
@@ -7,6 +7,32 @@ Tensors and all other Arrow entities.
 
 .. contents::
 
+Creating Tables
+===============
+
+Arrow supports tabular data in :class:`pyarrow.Table`: each column
+is represented by a :class:`pyarrow.Array` and tables can be created
```

Comment:
Internally, each column is a chunked array. But (non-chunked) arrays are also supported for constructing tables, for convenience.

Reply:
:+1: replaced `Array` with `ChunkedArray`